Simplified/NYPLBookDetailViewController.m

```diff
@@ -126,18 +139,92 @@ - (void)didSelectReadForBookDetailView:(__attribute__((unused)) NYPLBookDetailVi
    animated:YES];
 }
 
-- (void)didSelectTryAgainForBookDetailView:(__attribute__((unused)) NYPLBookDetailView *)detailView
+- (void)didSelectCitationsForBook:(NYPLBook *)book sender:(id)sender
 {
-  [[NYPLMyBooksDownloadCenter sharedDownloadCenter] startDownloadForBook:self.book];
+  //FIXME: add logic for launching citations here
```

Comment:
I am not sure citations should be a part of this pull request. As far as I know, the work on citations is not yet complete.